src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -122,33 +138,39 @@ export const BaseCourseSearchResults = ({
       >
         Your learners have been enrolled.
       </Toast>
-      <div>
-        <DataTable
-          columns={columns}
-          data={searchResults?.hits || []}
-          itemCount={searchResults?.nbHits}
-          isSelectable
-          pageCount={searchResults?.nbPages || 1}
-          pageSize={searchResults?.hitsPerPage || 0}
-          bulkActions={[{
-            buttonText: 'Enroll Learners',
-            handleClick: (selectedRows) => {
-              const courseRunKeys = selectedRows?.map(
-                (selectedRow) => selectedRow.original?.advertised_course_run?.key,
-              ) || [];
-              setSelectedCourseRuns(courseRunKeys);
-              setModalOpen(true);
-            },
-          }]}
-        >
-          <DataTable.TableControlBar />
-          <DataTable.Table />
-          <DataTable.TableFooter>
-            <DataTable.RowStatus />
-            <SearchPagination defaultRefinement={page} />
-          </DataTable.TableFooter>
-        </DataTable>
-      </div>
+      <DataTable
+        columns={columns}
+        data={searchResults?.hits || []}
+        itemCount={searchResults?.nbHits}
+        isSelectable
+        pageCount={searchResults?.nbPages || 1}
+        pageSize={searchResults?.hitsPerPage || 0}
+        bulkActions={[{
+          buttonText: ENROLL_TEXT,
+          handleClick: handleBulkEnrollClick,
+        }]}
+        additionalColumns={[
```

Comment:
didn't realize there was this idea of `additionalColumns`, how is it different from regular columns?

Reply:
It's a column that's not based on an accessor from the table data, basically. It has access to the row data so you can do what you want with it, but you can't really give it an accessor like you can with the other columns.